docs/installation/expansion.rst

```diff
@@ -65,12 +49,23 @@ The MetalK8s GUI provides a simple form for that purpose.
    - **SSH Key Path**: the path to the private key generated in
      :ref:`this procedure <Bootstrap SSH Provisioning>`
    - **Sudo required**: whether the SSH deployment will need ``sudo`` access
-   - **Roles/Workload Plane**: check this box if the new Node should receive
-     workload applications
-   - **Roles/Control Plane**: check this box if the new Node should run
-     control-plane services
-   - **Roles/Infra**: check this box if the new Node should run
-     infra services
+   - **Roles/Workload Plane**: if you select this box, you will remove all the
+     taints and enable the node role if Infra Box isn't selected.
+   - **Roles/Control Plane**: if you select this box, you enable both role
```

Comment:
enables both role and taint for master and etcd.

Reply:
`Roles/Control Plane: enable master and etcd services`